crates/bevy_asset/src/io/embedded/mod.rs

```diff
@@ -299,6 +299,73 @@ mod tests {
         assert_eq!(asset_path, Path::new("my_crate/foo/the/asset.png"));
     }
 
+    // A blank src_path removes the embedded's file path altogether.
+    #[test]
+    fn embedded_asset_path_from_local_crate_blank_src_path_questionable() {
```

Comment:
This one seems like it's testing `Path` more than `_embedded_asset_path` per se. So I generally don't like writing these tests when I know that conditions are handled by the infrastructure I'm using. I'd drop this test.

Reply:
Hmm yea I guess it's odd that this is a functional option. We could add a check that `src_path` isn't empty.